### PR TITLE
OM-175 - Vector search dashboard

### PR DESCRIPTION
**Pull request by mphanias**
initial checkin of vector search dashboard

### CODE REVIEW

File: config/grafana/dashboards/vectorsearch/vector_search_view.json [hunk 1]
```diff
@@ -0,0 +1,1919 @@
...
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Proximus View dashboard helps visualise indexed, query success/error in selected proximus cluster",
```
Comment (hev) [inline config/grafana/dashboards/vectorsearch/vector_search_view.json:56]: Let's use the name Aerospike Vector Search (AVS) instead of proximus.

**Review by hev — CHANGES_REQUESTED** [review]
This is looking good a few thoughts. I reviewed this by hooking it up to our demo which has 1M images. Click [this link](http://34.122.59.14:3000/d/fzUPYeJIk/aerospike-proximus-cluster-view?orgId=1&refresh=1m&var-DS_AEROSPIKE_PROMETHEUS=P07E1502634787998&var-job_name=proximus&var-cluster=ptrial-image-demo&var-instance=All&var-index=All&from=now-30d&to=now) on VPN to see it. 

1. I'd like to get a sense of how many vectors are in proximus in addition to the index size.
2. A metric for overall write rate into proximus is needed. We we're trying to see how much load we were putting on proximus when we were embedding the images and it was hard to figure out. Ideally we can just get a metric for puts into proximus
3. No matter what I do I can't seem to get the query rate to budge. 

Just a few initial thoughts.

**Review by thecloudslayer — APPROVED** [review]

**Review by hev — CHANGES_REQUESTED** [review]
A few more changes.

* I'd like to see storage (in GB) in the main panel
* reocmmending the lablel "Indexed Vectors" instead of Index Count. This is clearer
* The different query types are not clear to me

File: config/grafana/dashboards/vectorsearch/vector_search_view.json [hunk 2]
```diff
@@ -0,0 +1,2062 @@
...
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Aerospike Vector Service",
```
Comment (arrowplum) [inline config/grafana/dashboards/vectorsearch/vector_search_view.json:2058]: I think it is now no longer a service but a search

```suggestion
  "title": "Aerospike Vector Search",
```

Comment (hev) [reply config/grafana/dashboards/vectorsearch/vector_search_view.json]: Yep.

**Pull request merged**